code/datums/gamemodes/campaign/missions/mech_wars.dm

```diff
@@ -114,6 +114,12 @@
 	faction = FACTION_SOM
 	colors = list(ARMOR_PALETTE_GINGER, ARMOR_PALETTE_BLACK, VISOR_PALETTE_SYNDIE_GREEN)
 
+/obj/effect/landmark/campaign/mech_spawner/som/spawn_mech()
```

Comment:
This is part of the changes to give vehicles their own IFF signals instead of grabbing it from whoever the driver is. The default for the IFF is TGMC. My solution for the SOM mechs is just to append the SOM IFF signal to the mech through the spawner. I could use the spawner's `faction` var with the `GLOB.faction_to_iff` if you think it's better. Would support more factions automatically in the future.